sdk/spring/spring-cloud-azure-core/src/main/java/com/azure/spring/core/converter/AzureRequestRetryOptionsConverter.java

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.spring.core.converter;
+
+import com.azure.spring.core.properties.retry.StorageRetryProperties;
+import com.azure.spring.core.properties.retry.RetryProperties;
+import com.azure.storage.common.policy.RequestRetryOptions;
+import com.azure.storage.common.policy.RetryPolicyType;
+import org.springframework.core.convert.converter.Converter;
+
+import java.time.Duration;
+
+/**
+ * Converts a {@link StorageRetryProperties} to a {@link RequestRetryOptions}.
+ */
+public final class AzureRequestRetryOptionsConverter implements Converter<StorageRetryProperties, RequestRetryOptions> {
```

Comment:
Shouldn't this be defined in the spring-cloud-azure-service module?

Reply:
OK, so all the converters should be moved to the azure-service module?